packages/build/webpack/constants.js

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright (c) 2019 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0: http://www.apache.org/licenses/LICENSE-2.0
+ */
+const lkModule = process.env.LK_MODULE;
+const lkModuleContainer = process.env.LK_MODULE_CONTAINER;
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+
+// This path assumes the enlistment in labkey-ui-components is a sibling of the root of the LabKey enlistment.
+const labkeyUIComponentsRelPath = (lkModuleContainer ? "../../../../../" : "../../../../") + "labkey-ui-components/packages/components";
```

Comment:
Hmm.  So, by using the npm module for the webpack config, we make it slightly more difficult for anyone to deviate from this enlistment pattern.  I've now heard that several other people have their enlistments in labkey-ui-components under the root enlistment.  Can we maybe use another `process.env` setting for the relative (or absolute) path?  Or just a switch to say whether it's a sibling or a child?

Reply:
As we discussed, we are going to leave this as is for this PR but make sure to investigate how/if we can change this to be more flexible going forward (i.e. maybe by having this JS file get the path from something other then basing it on a relative path from the module dir).

Reply:
Maybe add a TODO?

Reply:
added